src/renderer/components/SelectMediaBlock.tsx

```diff
@@ -14,34 +21,45 @@ const SelectMediaBox = styled(Box)`
   }
 `;
 
-const ImportFileBox = styled(Box)`
-  display: flex;
-  flex-direction: column;
-  text-align: center;
-  justify-content: center;
-  align-items: center;
-  margin: 80px 0;
-`;
-
 interface Props {
-  mediaFilePath: string | null;
+  setMediaFileName: Dispatch<SetStateAction<string | null>>;
   setMediaFilePath: Dispatch<SetStateAction<string | null>>;
+  setIsAwaitingMedia: Dispatch<SetStateAction<boolean>>;
 }
 
-const SelectMediaBlock = ({ mediaFilePath, setMediaFilePath }: Props) => {
+const SelectMediaBlock = ({
+  setMediaFileName,
+  setMediaFilePath,
+  setIsAwaitingMedia,
+}: Props) => {
   const selectMedia: () => Promise<void> = async () => {
     const selectedMedia = await window.electron.requestMediaDialog();
+
+    if (selectMedia !== null) {
+      setIsAwaitingMedia(false);
+    }
+
     setMediaFilePath(selectedMedia);
+
+    const fileName = await extractFileNameWithExtension(selectedMedia);
+
+    setMediaFileName(fileName);
   };
 
   return (
     <SelectMediaBox onClick={selectMedia}>
-      <Typography fontWeight="bold">Video or Audio Base</Typography>
-      {mediaFilePath}
-      <ImportFileBox>
-        <AddIcon fontSize="large" />
-        <Typography fontWeight="bold">Import File or Drag</Typography>
-      </ImportFileBox>
+      <InnerBox>
+        <Stack
+          direction="column"
+          alignItems="center"
+          justifyContent="space-between"
+          sx={{ height: '150px' }}
+        >
+          <Typography fontWeight="bold">Drag and drop file here</Typography>
+          <Typography fontWeight="bold">or</Typography>
```

Comment:
This text should use p-300 style

Reply:
done in: https://github.com/patrickbrett/mlvet/pull/38/commits/724472e1594ea09a26fe28ffdedd877554a9e3ec